<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALPACAModelRegistration</class>
 <widget class="qMRMLWidget" name="ALPACAModelRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>426</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="pointDensity">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Registration</string>
      </attribute>
      <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>641</width>
         <height>141</height>
        </rect>
       </property>
       <property name="text">
        <string>Model registration</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Source Model: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="sourceModelSelector">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Target Model: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="targetModelSelector">
          <property name="nodeTypes">
           <stringlist>
            <string>vtkMRMLModelNode</string>
           </stringlist>
          </property>
          <property name="hideChildNodeTypes">
           <stringlist notr="true"/>
          </property>
          <property name="interactionNodeSingletonTag">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="skipScalingCheckBox">
          <property name="text">
           <string>Skip Scaling</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="ctkCollapsibleButton" name="CollapsibleButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>641</width>
         <height>171</height>
        </rect>
       </property>
       <property name="text">
        <string>Test subsampling pointclouds</string>
       </property>
       <widget class="QPlainTextEdit" name="subsampleInfo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>621</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="ctkSliderWidget" name="pointDensitySlider">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>30</y>
          <width>451</width>
          <height>21</height>
         </rect>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Point Density Adjustment</string>
        </property>
       </widget>
       <widget class="QPushButton" name="subsampleButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>621</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Run subsampling</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="runRegistrationButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>310</y>
         <width>631</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Run the algorithm.</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced Settings</string>
      </attribute>
      <widget class="ctkCollapsibleButton" name="CollapsibleButton_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>641</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Point Density</string>
       </property>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Point Density Adjustment</string>
        </property>
       </widget>
       <widget class="ctkSliderWidget" name="pointDensityAdvancedSlider">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>351</width>
          <height>20</height>
         </rect>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="ctkCollapsibleButton" name="CollapsibleButton_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>641</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string>Rigid Regisstration</string>
       </property>
       <widget class="ctkSliderWidget" name="normalSearchRadiusSlider">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>30</y>
          <width>351</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Normal Search Radius</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>FPFH Search Radius</string>
        </property>
       </widget>
       <widget class="ctkSliderWidget" name="FPFHSearchRadiusSlider">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>60</y>
          <width>351</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>3.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>261</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum Corresponding Point Distance</string>
        </property>
       </widget>
       <widget class="ctkSliderWidget" name="maximumCPDThreshold">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>90</y>
          <width>351</width>
          <height>31</height>
         </rect>
        </property>
        <property name="singleStep">
         <double>0.250000000000000</double>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>6.000000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum RANSAC Iterations</string>
        </property>
       </widget>
       <widget class="ctkDoubleSpinBox" name="maxRANSAC">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>120</y>
          <width>351</width>
          <height>31</height>
         </rect>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Poisson Point Subsample </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum ICP distance</string>
        </property>
       </widget>
       <widget class="ctkSliderWidget" name="ICPDistanceThresholdSlider">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>180</y>
          <width>351</width>
          <height>31</height>
         </rect>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="value">
         <double>1.500000000000000</double>
        </property>
       </widget>
       <widget class="QCheckBox" name="poissonSubsampleCheckBox">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>160</y>
          <width>86</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>FPFP Neighbor Count</string>
        </property>
       </widget>
       <widget class="ctkSliderWidget" name="FPFHNeighborsSlider">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>210</y>
          <width>351</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimum">
         <double>10.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
